src/audio.js

```diff
@@ -28,13 +28,12 @@ export class NMAudioManager {
         return this.muted;
     }
 
-    playNoMames() {
+    playNoMames(audionum) {
```

Comment:
You snuck that change in this PR :-P. Remember to split PRs to keep them to a single change!